docs/data/joy/components/input/input.md

```diff
@@ -166,6 +166,24 @@ You can use these to customize the component with both the `sx` prop and the the
 
 {{"demo": "InputVariables.js", "hideToolbar": true}}
 
+## Third-party library integration
+
+To integrate the Input component with 3rd party libraries, create a wrapper component to map props with 3rd party component APIs.
+
+Then, use the wrapper component as a value to the `slotProps.input.component` property of the Joy UI Input.
+
+### React imask
+
+[react-imask](https://github.com/uNmAnNeR/imaskjs) provides the vanilla JavaScript `IMaskInput` component, which enables multiple ways to format text.
```

Comment:
Maybe we should link here instead? I was a little confused when I landed on the `imaskjs` page. 

```suggestion
[react-imask](https://github.com/uNmAnNeR/imaskjs/tree/master/packages/react-imask) provides the `IMaskInput` component for complex formatting options.
```